<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrainingDetailMainPanel</class>
 <widget class="QWidget" name="TrainingDetailMainPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>867</width>
    <height>584</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>867</width>
    <height>584</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>867</width>
    <height>584</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>24</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="headerLayout">
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="AutomationCellProperty" name="aiModel" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>144</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>144</width>
         <height>70</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="AutomationCellProperty" name="createdOn" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>70</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="AutomationCellProperty" name="createdBy" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>70</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="promptLayout">
     <property name="spacing">
      <number>12</number>
     </property>
     <item>
      <widget class="QPushButton" name="prompt">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>115</width>
         <height>32</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent;
border: none;
font-family: Open Sans;
font-weight: 600;
font-size: 20px;
line-height: 28px;
color: #1a1a1a;</string>
       </property>
       <property name="text">
        <string>Prompt</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../../Resources/Resources.qrc">
         <normaloff>:/images/Automations/prompt.svg</normaloff>:/images/Automations/prompt.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="promptLabel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>52</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>52</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font-family: Open Sans;
font-size: 18px;
font-weight: 400;
line-height: 26px;
color: #1a1a1a;</string>
       </property>
       <property name="text">
        <string>Lorem ipsum dolor sit amet consectetur. Dignissim adipiscing enim sit tellus tempus nec. Sed erat massa volutpat urna. Consectetur nisi nullam sed fringilla ac ornare nunc nunc risus. See more</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="TrainingDetailTable" name="table" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 0px;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AutomationCellProperty</class>
   <extends>QWidget</extends>
   <header location="global">automationcellproperty.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TrainingDetailTable</class>
   <extends>QWidget</extends>
   <header>trainingdetailtable.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../../Resources/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
